hooks/versionCron.hook.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * This hook will run once a day (at midnight), check the current deployed version of the app, compare with the currently
+ * running version of the app, and send superadmins an email notifying them of the latest version with release notes, which will
+ * also reside in the github repo.
+ */
+const axios = require('axios')
+const cron = require('node-cron')
+const semver = require('semver')
+require('dotenv').config()
+
+module.exports = {
+  getFileFromGithub(url) {
+    const { GITHUB_TOKEN } = process.env
+    return axios({
+      method: 'get',
+      url,
+      headers: {
+        Authorization: `Bearer ${GITHUB_TOKEN}`,
+        Accept: 'application/vnd.github.v3.raw'
+      }
+    })
+  },
+
+  serverRepoURL(file) {
+    return `https://raw.githubusercontent.com/NTACT/transition-gradebook-server/master/${file}`
+  },
+
+  versionEmailTemplate(notes) {
+    return `
+      <p> There is a new version of Transition Gradebook that is available for installation. Please update your app to the latest version. </p>
```

Comment:
Ever use handlebars.js? We do similar things in other projects using that (not familiar enough with TGB to know if its used in this project). Worth checking out if interested, but not a huge deal (at least for me).